modules/misc/news/2025/09/2025-09-01_14-57-54.nix

```diff
@@ -0,0 +1,7 @@
+{
+  time = "2025-09-01T12:57:54+00:00";
+  condition = true;
```

Comment:
Don't really think we need a news entry for something that has a `mkRemovedOptionModule` tbh. But, if you are keen on it, at least gate it for user's who have the module enabled.